docs/user_guide/mitigation.rst

```diff
@@ -80,11 +81,408 @@ Reductions
 
 .. currentmodule:: fairlearn.reductions
 
+From a very high-level perspective the reductions techniques within Fairlearn
+enable unfairness mitigation on an arbitrary machine learning model with
+respect to a user provided fairness objective. For more information on the
+supported kinds of fairness refer to :ref:`constraints_binary_classification`
+and :ref:`constraints_regression`. All of the constraints currently supported
+by reductions methods are group fairness constraints.
+Note that the choice of fairness constraint is crucial to the outcome, and
+choosing an unsuitable constraint can cause or worsen harms on individuals or
+groups affected by a decision. For a broader discussion of fairness as a
+sociotechnical challenge and how to view Fairlearn in this context refer to
+:ref:`fairness_in_machine_learning`.
+
+The reductions approach is based on the idea in [#2]_ to reduce binary
+classification subject to fairness constraints to a sequence of cost-sensitive
+classification problems. As a result, the reductions methods in Fairlearn
+require only black-box access to a cost-sensitive classification algorithm,
+which does not need to have any knowledge of the desired definition of
+fairness or sensitive features. [#1]_ extends this to regression scenarios.
+
+From an API perspective this looks as follows in all situations
+
+>>> reduction = Reduction(estimator, constraints, **kwargs)  # doctest: +SKIP
+>>> reduction.fit(X_train, y_train, sensitive_features=sensitive_features)  # doctest: +SKIP
+>>> reduction.predict(X_test)  # doctest: +SKIP
+
+Fairlearn doesn't impose restrictions on the referenced :code:`estimator`
+other than the existence of :code:`fit` and :code:`predict` methods.
+At the moment, the :code:`estimator`'s :code:`fit` method also needs to
+provide a :code:`sample_weight` argument which the reductions techniques use
+to reweight samples.
+In the future Fairlearn will provide functionality to handle this even
+without a :code:`sample_weight` argument.
+
+Before looking more into the reductions approaches themselves this section
+examines the different constraints that are compatible with them. All of them
+are expressed as moments inheriting from :code:`ClassificationMoment` or
+:code:`LossMoment`, which in turn inherit from the base class :code:`Moment`.
```

Comment:
Not a fan of "...base class Moment. Moment's main purpose...."

Reply:
replaced second one with "It"